cafebabel/templates/home.html

```diff
@@ -39,4 +39,30 @@ <h2>Recently published</h2>
     {{ card(article) }}
   {% endfor %}
 {% endif %}
+  <div id=under-construction>
+    <div class=modal>
+    <button></button>
```

Comment:
Empty button with no text and no title?
Maybe some content could be more accessibility-friendly.